### PR TITLE
C: add MultiGet support

### PR DESCRIPTION
**Pull request by rdallman**

### CODE REVIEW

File: db/c_test.c [hunk 1]
```diff
@@ -505,6 +505,33 @@ int main(int argc, char** argv) {
...
   }
 
+  StartPhase("multiget");
+  {
+    const char* keys[3] = { "box", "foo", "notfound" };
+    const size_t keys_sizes[3] = { 3, 3, 8 };
+    char* vals[3];
+    size_t vals_sizes[3];
+    char* errs[3];
+    rocksdb_multi_get(db, roptions, 3, keys, keys_sizes, vals, vals_sizes, errs);
```
Comment (igorcanadi) [inline db/c_test.c]: let's also test rocksdb_muliti_get_cf() to increase code coverage

Comment (rdallman) [inline db/c_test.c]: can do

Comment (rdallman) [inline db/c_test.c]: done: https://github.com/facebook/rocksdb/pull/622/files#diff-ade08658a6ad910c75f0508bcbcd2671R814

**Comment by igorcanadi** [comment]
Thanks!

**Pull request merged**